src/setFocus.ts

```diff
@@ -1,7 +1,16 @@
 import { getFocusMerge } from './focusMerge';
 
-export const focusOn = (target: HTMLInputElement | HTMLFrameElement) => {
-  target.focus();
+type FocusOptionsType = {
```

Comment:
I have used a separate type from HTML focus options type & cherry picked `preventScroll` from the object instead of passing focusOptions as a whole to focus function is because later if we want to change it or introduce new options, we might do it without breaking changes. Hope that's okay.